### PR TITLE
Unified alarm panel icons everywhere

### PR DESCRIPTION
**Pull request by kukulich**
## Proposed change

Unified alarm panel icons everywhere

There should be same in alarm panel card and also in badges.

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [x] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [X] The code change is tested and works locally.
- [X] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/components/entity/ha-state-label-badge.ts [hunk 1]
```diff
@@ -136,25 +136,6 @@ export class HaStateLabelBadge extends LitElement {
...
-          return "hass:weather-night";
-        }
-        if (entityState.state === "armed_custom_bypass") {
-          return "hass:shield-home";
-        }
-        if (entityState.state === "triggered") {
-          return "hass:alert-circle";
-        }
-        // state == 'disarmed'
         return domainIcon(domain, entityState);
```
Comment (bramkragten) [inline src/components/entity/ha-state-label-badge.ts:139]:
```suggestion
```

Comment (bramkragten) [reply src/components/entity/ha-state-label-badge.ts:139]: Just use `stateIcon`

File: src/common/entity/alarm_panel_icon.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
+/** Return an icon representing a alarm panel state. */
+
+export const alarmPanelIcon = (state?: string) => {
+  switch (state) {
+    case "armed_away":
+    case "armed_vacation":
+      return "hass:shield-lock";
```
Comment (bramkragten) [inline src/common/entity/alarm_panel_icon.ts:7]:
```suggestion
      return "hass:shield-sun";
```

File: src/common/entity/alarm_panel_icon.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    case "armed_night":
+      return "hass:shield-moon";
+    case "armed_custom_bypass":
+      return "hass:security";
+    case "pending":
+      return "hass:shield-outline";
+    case "triggered":
+      return "hass:bell-ring";
+    default:
+      return "hass:shield-check";
```
Comment (bramkragten) [inline src/common/entity/alarm_panel_icon.ts:19]: This should be the `disarmed` state I think, or `shield-off`? We should add a different default, I think, maybe just `shield`.

File: src/common/entity/alarm_panel_icon.ts (hunk 2 above)
Comment (bramkragten) [reply src/common/entity/alarm_panel_icon.ts:7]: For vacation?

File: src/common/entity/alarm_panel_icon.ts [hunk 4]
```diff
@@ -0,0 +1,21 @@
+/** Return an icon representing a alarm panel state. */
+
+export const alarmPanelIcon = (state?: string) => {
+  switch (state) {
+    case "armed_away":
```
Comment (bramkragten) [inline src/common/entity/alarm_panel_icon.ts:5]:
```suggestion
    case "armed_away":
      return "hass:shield-lock";
```

File: src/common/entity/alarm_panel_icon.ts (hunk 2 above)
Comment (kukulich) [reply src/common/entity/alarm_panel_icon.ts:7]: Make sense now, when we use `moon` for night. Or we can use `mdi:shield-airplane`.

**Review by bramkragten — DISMISSED** [review]

File: src/common/entity/alarm_panel_icon.ts (hunk 2 above)
Comment (bramkragten) [reply src/common/entity/alarm_panel_icon.ts:6]: Yeah airplane would work well too!

Comment (kukulich) [reply src/common/entity/alarm_panel_icon.ts:6]: @bramkragten Should I push the `airplane` icon? I can do it right now.

Comment (bramkragten) [reply src/common/entity/alarm_panel_icon.ts:6]: yeah sure! :-)

Comment (kukulich) [reply src/common/entity/alarm_panel_icon.ts:6]: Done.

**Review by bramkragten — APPROVED** [review]

**Pull request merged**